internal/output/output_result.go

```diff
@@ -401,10 +402,16 @@ func processPackage(vulnPkg models.PackageVulns) PackageResult {
 
 	packageFixedVersion := calculatePackageFixedVersion(vulnPkg.Package.Ecosystem, regularVulnList)
 
+	installVersion := vulnPkg.Package.Version
+	// If the ecosystem is not set, it's most likely from Git
+	if installVersion == "" && vulnPkg.Package.Ecosystem == "" {
+		installVersion = results.GetShortCommit(vulnPkg.Package.Commit)
+	}
+
 	packageResult := PackageResult{
 		Name:             vulnPkg.Package.Name,
```

Comment:
Can you also check if name is empty, but repo is not empty, set name to repo?

Reply:
What do you mean by repo? we don't have repo information in `PackageVulns.PackageInfo` though.

Reply:
Ah ok we haven't passed that through, let's ignore this for now.